docs/_includes/linter-breadcrumb.html

```diff
@@ -11,18 +11,6 @@
     </li>
     {%- if (page.url | slice: 0, 7) == '/rules/' %}
     {%- assign url_segments = page.url | split: '/' %}
-    {%- if url_segments | first | abs %}
-    <li class="glue-breadcrumbs__item" aria-level="2">
-      <a class="glue-breadcrumbs__link" href="{{ site.url }}/rules/">
-        Rule Documentation
-      </a>
-    </li>
-    <li class="glue-breadcrumbs__item" aria-level="2">
-      <a class="glue-breadcrumbs__link" href="{{ site.url }}/rules/core/">
-        Core rules
-      </a>
-    </li>
```

Comment:
This was unnecessary because the following `{%- for segment in url_segments` logic does this in a more robust way.